### PR TITLE
Add a WebGPU example

### PR DESCRIPTION
**Pull request by toji**
Nice page! Wanted to contribute a WebGPU example to sit alongside the WebGL one.

### CODE REVIEW

File: examples/webgpu/script.js [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+  device,
+  format: navigator.gpu.getPreferredCanvasFormat(),
+  alphaMode: 'premultiplied',
+});
+
+// Set up the vertex and fragment shaders.
+const module = device.createShaderModule({
+  code: `
+    // Define the four corners of the square being rendered
+    var<private> pos : array<vec2f, 4> = array<vec2f, 4>(
```
Comment (greggman) [inline examples/webgpu/script.js:18]:
```suggestion
    var<private> pos = array(
```

**Review by greggman — COMMENTED** [review]
best practices = using labels?

File: examples/webgpu/index.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  <title>Bouncy Ball - Comparing Web Animation Techniques - WebGPU</title>
+  <link rel="stylesheet" href="../shared-styles.css">
+  <style>
+  body,
+  html,
+  #canvas {
+    height: 100%;
+    width: 100%;
+    margin: 0;
+  }
```
Comment (greggman) [inline examples/webgpu/index.html:14]: This is incomplete CSS. You can see the issue here: 

https://sparkbox.github.io/bouncy-ball/examples/webgl/index.html

There's a scrollbar.  Add the `display: block;` and the canvas goes from `inline` to `block` and the scrollbar disappears


```suggestion
    display: block;
  }
```

File: examples/webgpu/script.js (hunk 1 above)
Comment (toji) [reply examples/webgpu/script.js:18]: I didn't know about this bit of syntactic sugar! Thanks!

Comment (toji) [reply examples/webgpu/script.js]: Done

File: examples/webgpu/index.html (hunk 2 above)
Comment (toji) [reply examples/webgpu/index.html:15]: Done

**Comment by toji** [comment]
Thanks! Addressed your feedback and added labels where appropriate.

File: site/js/updatePanes.js [hunk 3]
```diff
@@ -93,6 +93,10 @@ function isCompatible(selectedId) {
   if (selectedId === 'p5') {
     return Modernizr.webgl;
   }
+  if (selectedId === 'webgpu') {
+    // WebGPU is still in development on some browsers.
```
Comment (bryanbraun) [inline site/js/updatePanes.js:97]: Nice!

**Review by bryanbraun — APPROVED** [review]
Love it! Thanks so much for adding this @toji and @greggman. 🙇‍♂️

**Pull request merged**